srcjs/shinyapp.js

```diff
@@ -632,7 +632,12 @@ var ShinyApp = function() {
 
   addMessageHandler('config', function(message) {
     this.config = {workerId: message.workerId, sessionId: message.sessionId};
+    // != is on purpose (instead of !==) because it checks for both null and undefined
+    // (see here: http://stackoverflow.com/a/21273362/6174455)
+    /*eslint-disable */
     if (message.user != null) exports.user = message.user;
+    /*eslint-enable */
+    $(document).trigger('shiny:sessionInit');
```

Comment:
For consistency with other events, this should be all lower case. I think that `shiny:sessioninitialized` would be better, since it indicates that it happens after the initialization.